compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/KotlinPoetUtils.kt

```diff
@@ -229,6 +233,10 @@ public fun KotlinType.asTypeName(): TypeName {
  *                      resolve type arguments.
  */
 @ExperimentalAnvilApi
+@Suppress("DeprecatedCallableAddReplaceWith")
+@Deprecated(
+  "Don't rely on descriptors and make the code agnostic to the underlying implementation."
+)
 public fun KotlinType.asTypeNameOrNull(
```

Comment:
I noticed this one mentions descriptors but the deprecation message for `KotlinType.asTypeName()` mentions PSI. Should one be swapped to match?